### ISSUE TITLE
[Tooltip] style: design feedback

### ISSUE THREAD

**Issue by akdetrick** [narmi/design_system#421]
BLOCKED by:
- #333 

- [ ] make the tip of the arrow rounded if possible (can we use SVG?)
- [ ] add an `offset` prop to control distance from trigger (with a sensible default)

**Comment by akdetrick** [comment]
duplicate of #858
